presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraDistributed.java

```diff
@@ -72,7 +73,10 @@ public void testRenameColumn()
     @Override
     public void testInsert()
     {
-        // Cassandra connector currently does not support insert
+        execute("DROP TABLE IF EXISTS tpch.table_insert");
+        execute("CREATE TABLE tpch.table_insert (id int PRIMARY KEY)");
+        execute("INSERT INTO tpch.table_insert (id) VALUES (1)");
```

Comment:
Cassandra connector does not support `CREATE TABLE` (`CREATE TABLE AS` is only supported) and `INSERT SELECT` currently, so I'm going to copy the base of `testInsert()` and modify it. 
I think the all supported Cassandra datatypes with `TestCassandraIntegrationSmokeTest` is little difficult. The first reason is no support for `CREATE TABLE`. The second reason is that `CREATE TABLE AS` generates hidden id column, but inserting value to hidden column is not allowed.

Do you have any suggestion about that? I've added `execute(String sql)` method in `TestCassandraDistributed` to create table with `cql` and avoid above situation.  If we can use `cql` in smoketest, the test is easy. But I think it isn't good approach.